---
title: IBMQBackendValueError
description: API reference for qiskit.providers.ibmq.IBMQBackendValueError
in_page_toc_min_heading_level: 1
python_api_type: exception
python_api_name: qiskit.providers.ibmq.IBMQBackendValueError
---

<span id="qiskit-providers-ibmq-ibmqbackendvalueerror" />

# qiskit.providers.ibmq.IBMQBackendValueError

<Class id="qiskit.providers.ibmq.IBMQBackendValueError" isDedicatedPage={true} github="https://github.com/qiskit/qiskit-ibmq-provider/tree/stable/0.12/qiskit/providers/ibmq/exceptions.py" signature="IBMQBackendValueError(*message)" modifiers="exception">
  Value errors raised by the backend modules.

  Set the error message.

  ### with\_traceback

  <Function id="qiskit.providers.ibmq.IBMQBackendValueError.with_traceback" signature="with_traceback()">
    Exception.with\_traceback(tb) â€“ set self.\_\_traceback\_\_ to tb and return self.
  </Function>
</Class>

